### PR TITLE
feature/CD-106: Label component

### PR DESCRIPTION
**Pull request by gusartus**

### CODE REVIEW

File: src/common/components/Label/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import styles from './label.scss';
+
+interface IProps {
+	color?: string;
+	style?: object;
+	text: string;
+}
+
+const Label: React.SFC<IProps> = ({color, text, style}) => (
```
Comment (CodeAnnihilator) [inline src/common/components/Label/Label.tsx]: SFC is deprecated. Use FC instead

**Comment by CodeAnnihilator** [comment]
image
it doesn't look like a label

**Comment by gusartus** [comment]
> image
> it doesn't look like a label

Because you put that components inside parent, which has flex-direction: column;
All flex-items (our label too) will be automatically resized to take all available space.
And we can see that on your screenshot

**Comment by CodeAnnihilator** [comment]
image
this is default look that we want to have. We don't want it to be stretched over the page

File: src/common/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		isActive: false,
+		onToggle: () => false,
+	};
+
+	public state = {
+		checkboxActive: this.props.isActive,
+	};
+
+	private readonly checkboxChangeHandler = () => {
+		this.setState({checkboxActive: !this.state.checkboxActive});
```
Comment (CodeAnnihilator) [inline src/common/components/Checkbox/Checkbox.tsx]: use setState as a function

File: src/common/components/Label/label.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@import 'styles/colors/scss';
```
Comment (CodeAnnihilator) [inline src/common/components/Label/label.scss]: may be colors.scss?

**Comment by CodeAnnihilator** [comment]
image
im wasting time on it. Please, check you PR properly

File: src/common/components/Label/label.scss (hunk 3 above)
Comment (gusartus) [reply src/common/components/Label/label.scss]: Fixed. I have no idea how this happened

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**